avm/res/databricks/workspace/README.md

```diff
@@ -0,0 +1,1473 @@
+
+- Required: Yes
+- Type: string
+
+### Parameter: `customerManagedKeyManagedDisk.keyVersion`
+
+The version of the customer managed key to reference for encryption. If not provided, using 'latest'.
+
+- Required: No
+- Type: string
+
+### Parameter: `customerManagedKeyManagedDisk.rotationToLatestKeyVersionEnabled`
+
+Indicate whether the latest key version should be automatically used for Managed Disk Encryption. Enabled by default.
+
+- Required: No
+- Type: bool
+
+### Parameter: `customerManagedKeyManagedDisk.userAssignedIdentityResourceId`
+
+User assigned identity to use when fetching the customer managed key. Required if no system assigned identity is available for use.
+
+- Required: No
+- Type: string
+
+### Parameter: `customPrivateSubnetName`
+
+The name of the Private Subnet within the Virtual Network.
+
+- Required: No
+- Type: string
+- Default: `''`
+
+### Parameter: `customPublicSubnetName`
+
+The name of a Public Subnet within the Virtual Network.
+
+- Required: No
+- Type: string
+- Default: `''`
+
+### Parameter: `customVirtualNetworkResourceId`
+
+The resource ID of a Virtual Network where this Databricks Cluster should be created.
+
+- Required: No
+- Type: string
+- Default: `''`
+
+### Parameter: `diagnosticSettings`
+
+The diagnostic settings of the service.
+
+- Required: No
+- Type: array
+
+**Optional parameters**
+
+| Parameter | Type | Description |
+| :-- | :-- | :-- |
+| [`eventHubAuthorizationRuleResourceId`](#parameter-diagnosticsettingseventhubauthorizationruleresourceid) | string | Resource ID of the diagnostic event hub authorization rule for the Event Hubs namespace in which the event hub should be created or streamed to. |
+| [`eventHubName`](#parameter-diagnosticsettingseventhubname) | string | Name of the diagnostic event hub within the namespace to which logs are streamed. Without this, an event hub is created for each log category. For security reasons, it is recommended to set diagnostic settings to send data to either storage account, log analytics workspace or event hub. |
+| [`logAnalyticsDestinationType`](#parameter-diagnosticsettingsloganalyticsdestinationtype) | string | A string indicating whether the export to Log Analytics should use the default destination type, i.e. AzureDiagnostics, or use a destination type. |
+| [`logCategoriesAndGroups`](#parameter-diagnosticsettingslogcategoriesandgroups) | array | The name of logs that will be streamed. "allLogs" includes all possible logs for the resource. Set to '' to disable log collection. |
+| [`marketplacePartnerResourceId`](#parameter-diagnosticsettingsmarketplacepartnerresourceid) | string | The full ARM resource ID of the Marketplace resource to which you would like to send Diagnostic Logs. |
+| [`metricCategories`](#parameter-diagnosticsettingsmetriccategories) | array | The name of logs that will be streamed. "allMetrics" includes all possible logs for the resource. Set to '' to disable log collection. |
```

Comment:
Readme and json now outdated. Please make sure to regenerate the files for every change and this and all other PRs. Otherwise the static tests will fail.

Reply:
done